src/test_l1_dsDisplay.c

```diff
@@ -560,12 +560,12 @@ void test_l1_dsDisplay_negative_dsGetEDID(void) {
     // Step 01: Call dsGetEDID() without initializing the display sub-system
     result = dsGetEDID(displayHandle, edid);
     UT_LOG("\n In %s Return value: [%d]\n", __FUNCTION__, result);
-    UT_ASSERT_EQUAL(result, dsERR_NOT_INITIALIZED);
+    UT_ASSERT_EQUAL_FATAL(result, dsERR_NOT_INITIALIZED);
 
     // Step 02: Initialize the display sub-system
     result = dsDisplayInit();
     UT_LOG("\n In %s Return value: [%d]\n", __FUNCTION__, result);
-    UT_ASSERT_EQUAL(result, dsERR_NONE);
+    UT_ASSERT_EQUAL_FATAL(result, dsERR_NONE);
 
     // Step 03: Obtain a display device handle
     for (int i = 0; i < sizeof(kSupportedPortTypes) / sizeof(kSupportedPortTypes[0]); i++) {
```

Comment:
Can we use the braces ( sizeof(kSupportedPortTypes) / sizeof(kSupportedPortTypes[0]) ) to have better operator precedence?